Client/Frontend/Browser/TopSitesManager.swift

```diff
@@ -49,12 +49,37 @@ struct TopSitesHandler {
         }
     }
     
+    static func writeTopSites(topSites: [TopSite]) {
```

Comment:
Would be nice to rename it to be more clear that its for widgets only. 
Ex. writeWidgetTopSites(...